ci/eval/default.nix

```diff
@@ -186,7 +210,22 @@ let
 
         # Make sure the glob doesn't break when there's no files
         shopt -s nullglob
```

Comment:
With the addition of `jq` into the pipeline, is this even needed anymore? Like, I’m pretty sure jq will error if it’s given no input.

Reply:
Ah, indeed, didn't notice the stats removal.